sdcm/nemesis/__init__.py

```
@@ -5259,23 +5267,26 @@ def disrupt_kill_mv_building_coordinator(self):
-                    )
-                except Exception as error:  # pylint: disable=broad-except
-                    self.log.error("Failed creating a materialized view: %s", error)
-                    raise
+            try:
+                create_materialized_view_for_random_column(session, ks_name, base_table_name, view_name)
+                wait_materialized_view_building_tasks_started(session, ks_name, view_name)
+            except ViewFinishedBuildingException:
+                drop_materialized_view(session, ks_name, view_name)
+                raise UnsupportedNemesis(
+                    f"Skip nemesis because view {ks_name}.{view_name} has already finished building"
+                )
```

Ruff (0.15.18)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)